app/src/androidTest/java/com/example/henzoshimada/feeltrip/FeelTripTests.java

```diff
@@ -67,12 +67,11 @@ public void testGetMoodTask(){
         assertEquals("test getMood", "user1", moods.get(0).getUser());
     }
 
-    // does not pass
     public void testEditTask() throws DescriptionTooLongException {
```

Comment:
Yes, that will be internally handled, this is just a mood event's specs (mood events won't ever change users)